angular/projects/admin-nrpti/src/app/records/records-list/records-list.component.ts

```diff
@@ -433,6 +454,29 @@ export class RecordsListComponent implements OnInit, OnDestroy {
     }
   }
 
+  /**
+   * Builds an array of project names, for project filters that are enabled/selected.
+   *
+   * @param {object} projects changes.projects object
+   * @returns {string[]} array of project names
+   * @memberof RecordsListComponent
+   */
+  getProjectsFilterArray(projects: object): string[] {
+    if (!projects) {
+      return [];
+    }
+
+    const projectsQueryParam: string[] = [];
+
+    for (const projectName of Object.keys(projects)) {
```

Comment:
You can use `for...in` to enumerate over an object's properties instead of 'for...of` and then you don't need to use `Object.keys()`.

Reply:
I think I've read somewhere that for...in is not preferred, as it has to make a bunch of extra calls and will grab inherited values and doesn't provide function scope.  (Obviously this use case is super simple, so it probably doesn't matter either way).  I'd probably be inclined to leave it as is, unless my memory is totally wrong

Reply:
Totally fine, both work well.